Sources/Jetworking/Extensions/URLSessionConfiguration+Extension.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+extension URLSessionConfiguration {
+    class func background(
```

Comment:
Why are you using class func instead of static?

Reply:
Wanted to be as close as possible to the original `.background` function. In my opinion we could leave it as is due to the fact that `URLSessionConfiguration` is an `NSObject`.